app/models/task.py

```diff
@@ -3,4 +3,31 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title =  db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable = True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'))
+
+    def convert_complete(self):
+        if self.completed_at == None:
+            complete = False
+        else:
+            complete = True
+        return complete
```

Comment:
This can be simplified to:

```suggestion
    def convert_complete(self):
        return self.completed_at != None
```